src/django_components/template.py

```diff
@@ -62,3 +75,363 @@ class MyTemplate(Template):
         template = maybe_cached_template
 
     return template
+
+
+########################################################
+# PREPARING COMPONENT TEMPLATES FOR RENDERING
```

Comment:
This section contains the updated logic for loading  component templates from files.